apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx

```diff
@@ -37,6 +40,12 @@ const ContributorsList = ({
 					{contributors.map((contributor) => (
 						<li key={contributor.id}>
 							<ContributorCard
+								activityLog={
+									activityLogs.find(
+										(activityLog) =>
+											Number(activityLog.contributorId) === contributor.id,
+									)?.commitsNumber
```

Comment:
maybe we can group it by contributor id inside useMemo outside of jsx and then access it here by contributor id?